package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "engines": {
```

Comment:
what is it ?

Reply:
It needs to know what version of npm we used. For example i didn't know what version we are using until you told me.

Reply:
https://docs.npmjs.com/files/package.json#engines

Reply:
you mean nodejs, ok, thx